src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -81,70 +84,67 @@ public class Button : Control
+                instance.iconPadding.CopyFrom((Extents)newValue);
                 instance.UpdateUIContent();
             }
         },
         defaultValueCreator: (bindable) =>
         {
             var instance = (Button)bindable;
-            return instance.Style?.IconPadding;
+            return instance.iconPadding;
         });
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static readonly BindableProperty TextPaddingProperty = BindableProperty.Create(nameof(TextPadding), typeof(Extents), typeof(Button), null, propertyChanged: (bindable, oldValue, newValue) =>
         {
             var instance = (Button)bindable;
-            if (null != newValue && null != instance.Style?.TextPadding)
+            if (null != newValue)
             {
-                instance.Style.TextPadding.CopyFrom((Extents)newValue);
+                if (null == instance.textPadding) instance.textPadding = new Extents(instance.OnTextPaddingChanged, 0, 0, 0, 0);
+                instance.textPadding.CopyFrom((Extents)newValue);
                 instance.UpdateUIContent();
             }
         },
         defaultValueCreator: (bindable) =>
         {
             var instance = (Button)bindable;
-            return instance.Style?.TextPadding;
+            return instance.textPadding;
         });
 
+        private ImageView overlayImage;
+        private TextLabel buttonText;
+        private ImageView buttonIcon;
+
         private EventHandler<StateChangedEventArgs> stateChangeHander;
 
+        private bool isSelectable;
         private bool isSelected = false;
         private bool isEnabled = true;
         private bool isPressed = false;
-
-        /// <summary>
-        /// Button's overlay image part.
-        /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected ImageView ButtonOverlayImage { get; set; }
-
-        /// <summary>
-        /// Button's text part.
-        /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected TextLabel ButtonText { get; set; }
-
-        /// <summary>
-        /// Button's icon part.
-        /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected ImageView ButtonIcon { get; set; }
+        private StringSelector translatableTextSelector = new StringSelector();
+        private StringSelector textSelector = new StringSelector();
+        private ColorSelector textColorSelector = new ColorSelector();
+        private FloatSelector pointSizeSelector = new FloatSelector();
+        private StringSelector iconURLSelector = new StringSelector();
+        private IconOrientation iconRelativeOrientation = IconOrientation.Left;
+        private Extents iconPadding;
+        private Extents textPadding;
```

Comment:
I think if there are same properties in Style, we can use them instead declaring them in Button. (to reduce size)
What do you think?